SGIS step record.txt

```diff
@@ -0,0 +1,25 @@
+when you run vagrant up fro the first time, if running setup.sh is not able to add extensions, please do the following manually after vagrant up:
```

Comment:
It should install the extensions the first time around, if it does then this file can be deleted. If it does not, the setup script should be changed.